packages/utils/src/read-file-or-url.ts

```diff
@@ -30,7 +30,8 @@ export async function readFileWithCache<T>(
   cache: KeyValueCache,
   config?: ReadFileOrUrlOptions
 ): Promise<T> {
-  const actualPath = isAbsolute ? filePath : resolve(process.cwd(), filePath);
+  const { readFile, stat } = await import('fs-extra');
```

Comment:
Why do we need that?

Reply:
IDK, this appeared while merging `master` branch to my local repo. Also this is your code https://github.com/Urigo/graphql-mesh/commit/718e7a165afe07f3a5897d066e01ce8a72f5779e#diff-94c8b93ac0f074be287562547a0d4595fece90657b814ae0f18a63ec36a52b05R29

Reply:
Yes we just removed `fs-extra`. I thought you needed soemthing specifically from fs-extra :)

Reply:
Could you remove that line? I cannot do that because I think you didn't select `allow maintainers to push the branch` option Thanks @germankrause